app/assets/javascripts/sequence_show.js

```diff
@@ -1,7 +1,10 @@
 function init_sequence_show(data) {
 
     // set up the fancy tree
-    initLineageTree(data.tree);
+    initD3TreeView(data.tree, "#lineage-tree");
```

Comment:
I think the LCA isn't highlighted...


_[Original comment](https://github.ugent.be/unipept/unipept/pull/528#discussion_r3470) by @bmesuere on Mon Nov 09 2015 at 22:14._